src/isPalindrome.js

```diff
@@ -0,0 +1,12 @@
+export default function isPalindrome(string) {
+
+  if (typeof string !== 'string') {
+    return 'Input is not a string'
+  }
+  var word = string.replace(/\W+/g, "").toLowerCase();
+  var arrStr = word.split('');
```

Comment:
Nice, but just like the line above this, you can do multiple operations on a single line. For example, you could do var reversed word = word.split().reverse().join()